### DISCUSSION TITLE
Future plans for platypus

### DISCUSSION THREAD

**Discussion by Astranix593** [WangYihang/Platypus#244] [General]
Hey @WangYihang, I'm curious what your plans for platypus are in the coming weeks, what features are soon to be supported, and what changes are being made. This project has sparked my interest and I'd love to contribute if you need help with anything (although my understanding of golang and nodeJS are very minimal)

**Comment by WangYihang** [comment]
Thanks for your attention to this matter, I will form a roadmap and discuss with you guys soon.

**Reply by Astranix593** [reply]
Hey there, no rush to you as I'm aware you're probably busy, but do you have an ETA on the roadmap? Thanks.

**Reply by WangYihang** [reply]
Hi there, thanks for checking in! No worries at all—I’ll make sure to provide a version by the end of this year.

**Reply by Astranix593** [reply]
What's the current ETA on this?? You said EOY and it's now almost february. No rush!!
